### ISSUE TITLE
Permission error reading HDFS in a dask cluster environment

### ISSUE THREAD

**Issue by rjk-lm95** [intake/filesystem_spec#292]
Hi,

I have a dask-gateway cluster setup on YARN and am trying to read a CSV file on HDFS.

Everything works fine when executing locally on the client
`import dask.dataframe as dd`
`df = dd.read_csv('webhdfs://namenode.example.com/path/csvfile.csv, 
storage_options={'kerberos'=True})`
`df.tail()` 

However when I initiate a dask cluster, and execute the same piece of code and it tries to execute on a dask worker, it throws a **PermissionError:** 

It appears that the delegation token isn't being loaded correctly from containing environment.

I'd appreciate any inputs or thoughts on this.

**Comment by martindurant** [comment]
Is the permission error happening in your local environment of within one of the workers?

**Comment by rjk-lm95** [comment]
It is occurring on the worker. I see a PermissionError() in the worker log.

**Comment by martindurant** [comment]
@jcrist , any thoughts on kerberos in yarn *for HTTP/spnego* ?
@rjk-lm95 , any reason you don't use HDFS, as opposed to webHDFS? Is this because your client is outside the cluster?

**Comment by rjk-lm95** [comment]
@martindurant That is correct. The client is outside the Hadoop cluster. So I'm trying to use webhdfs.
